Resources/Prototypes/Entities/buildings/gravity_generator.yml

```diff
@@ -0,0 +1,24 @@
+- type: entity
+  id: gravityGenerator
```

Comment:
This needs to follow the new [YAML naming guidelines](https://hackmd.io/Awj5TvdXTaOmhU2XZOsdQg).